the-9-exercises/src/App.jsx

```diff
@@ -1,14 +1,23 @@
-
-import './App.css'
+import List from "./components/List";
 
 function App() {
-
+  const arrayOfItems = [
+    "dog  ",
```

Comment:
Why do elements have spaces at the end??

Reply:
Same question here. If you are adding spaces for styling purposes you may achieve that with CSS instead. Nevertheless, you may not need the extra space